packages/backend/src/controllers/organizationController.ts

```diff
@@ -69,7 +69,7 @@ export class OrganizationController extends BaseController {
             status: 'ok',
             results: await this.services
                 .getOrganizationService()
-                .get(req.user!),
+                .get(req.account!),
```

Comment:
is it possible this is somehow not set ? for example, when using the CLI, using PAT, etc...

Reply:
I think we're catching it in all the places except for SCIM. PAT wraps the user in account after resolving the user:
https://github.com/lightdash/lightdash/blob/b987166c2fa7deb532a30c6c01f46213c2a9f6fa/packages/backend/src/controllers/authentication/middlewares.ts#L106-L118